src/Geno/C++/Auxiliary/jsonSerializer.cpp

```diff
@@ -15,35 +15,45 @@
  * 3. This notice may not be removed or altered from any source distribution.
  */
 
-#pragma once
-#include "GUI/Modals/IModal.h"
+#include "jsonSerializer.h"
```

Comment:
Wrong capitalization. Both the file and the class should be named `JSONSerializer`

Reply:
Okay